documentation/development/docker.md

```diff
@@ -0,0 +1,79 @@
+# Docker
+
+This describes how to create an environment for development using Docker.
+
+**This is still experimental, using [Vagrant](../installation.md) is much better tested and supported!**
```

Comment:
But if this works well for a few people with no issues, I'd like us to remove all code + documentation for the Vagrant version and standardize on this.

Reply:
Is it ok to push this to another story? That would:
1. allow more people to try this internally and provide feedback;
2. give use some time to improve the documentation structure a bit first: IMO it's conflating installation with the development setup.

Reply:
Yes definitely.